### ISSUE TITLE
Camera On/Off notification not showing

### ISSUE THREAD

**Issue by Ingrido** [BartoszCichecki/LenovoLegionToolkit#958]
### Rules

- [X] I made myself familiar with the README.
- [X] I checked closed issues for similar ones and did not find a similar problem mentioned before.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

2.16.2

### OS

Windows 11 22H2 build 22621.2215

### Device

Legion Pro 7 16IRX8H

### BIOS version

KWCN38WW

### What's wrong?

When toggling the physical camera switch, no camera on/off notification is popping up

### How to reproduce the bug?

Toggle the physical camera switch

### What is the behavior that you expected?

The notification showing if the camera is on/off would appear

### Logs

[log_2023_09_09_21_48_42.txt](https://github.com/BartoszCichecki/LenovoLegionToolkit/files/12566751/log_2023_09_09_21_48_42.txt)

**Comment by BartoszCichecki** [comment]
I dont see any camera on/off events in the log you sent so there is no notification to show. I also don't see any unhandled events.

Check that the switch is actually working and that you have all drivers installed.

LLT won't shown any notifications if the EC doesn't send an event.

**Comment by Ingrido** [comment]
How strange, the notification appears to be showing now, I did try reinstalling LLT yesterday but it was still not popping up so I made this post... so I guess it's fine now and I'll close this. Sorry for the bother.

But I definitely did toggle the switch during the logging and it was functioning fine just without the notification, as I tested using the Camera app, when turned off there would be no camera feed and when on there would be.
